src/locales/en.json

```diff
@@ -477,28 +483,28 @@
     "wasExportedAlert": "The report for this shift was already exported. This shift cannot be edited anymore.",
     "table": {
       "pastShiftsTitle": "Past shifts",
-      "pastShiftsHint": "Clocked shifts and shifts manually entered with past dates are considered as \"reviewed\".",
+      "pastShiftsHint": "Clocked shifts and manually entered shifts are considered as 'reviewed' if they are past.",
       "futureShiftsTitle": "Scheduled shifts",
       "futureShiftsHint": "All shifts scheduled for the future.",
       "tagsNotes": "Tags/Notes"
     },
     "errors": {
       "startedBeforeStopped": "The start must be before the end of the shift.",
-      "maximumWorktimePerDay": "The maxium allowed worktime ammount is 10 hours per day.",
+      "maximumworking timePerDay": "The maxium allowed working time amount is 10 hours per day.",
       "workingOnSundays": "It is not allowed to work on a sunday.",
-      "workingOnHolidays": "It is just allowed to book a holiday shift on a holiday.",
-      "exclusiveVacation": "It is not allowed to combinate vaction shifts with other shifts on a day.",
-      "exclusiveSick": "It is not allowed to combinate sick shifts with other shifts on a day.",
-      "overlappingStarted": "The started value is in the time of an already existing reviewed shift.",
-      "overlappingStopped": "The stopped value is in the time of an already existing reviewed shift.",
-      "eightTwentyRule": "Due to internal rules of the Frankfurt University it is not recommended to work before 8 am and after 20om.",
-      "autoWorktimeCutting": "On this workday are mandatory breaktimes missing. Clock will cutt the missing breaktimes out of the worktime automatically."
+      "workingOnHolidays": "You can only book 'holiday' shifts on a holiday.",
+      "exclusiveVacation": "You can not combine 'vacation' shifts with other shifts on the same day.",
+      "exclusiveSick": "You can not combine 'sick' shifts with other shifts on the same day.",
+      "overlappingStarted": "The start time is within the period of an existing and reviewed shift.",
+      "overlappingStopped": "The end time is within the period of an already existing and reviewed shift.",
+      "eightTwentyRule": "As a rule, student workers should not be obliged to work before 8 am and after 20pm.",
```

Comment:
Wir sollten hier einheitlich '8am' und '20pm' oder '8 am' und '20 pm' schreiben. Ggf sogar 'AM' und 'PM' verwenden?

Reply:
20 PM ist eigentlich redundant, oder? 😉 

Wie wäre es mit 8:00 und 20:00? Das ist eindeutig international verständlich - in Clock wird sowieso das 24h-Format verwendet.

Reply:
mache ich mal so